packages/lit-virtualizer/test/helpers.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * Use this after rendering, resizing or scrolling to `await` the
+ * reflow necessary before validating events and visibility changes.
+ */
+export async function wait(n=0) {
+    await new Promise(resolve => requestAnimationFrame(resolve));
+    return new Promise(resolve => setTimeout(resolve, n));
+};
+
+/**
+ * This solution was inspired to address the issue described in the following links:
+ * https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded
+ * https://stackoverflow.com/questions/9025095/how-can-i-test-uncaught-errors-in-mocha
+ * 
+ * Before/After parameters are given to the function to ensure there are no user-error
+ * cases where teardown is forgotten/skipped.
+ */
+export function ignoreBenignErrors(before, after) {
+    ignoreWindowErrors(before, after, /^(ResizeObserver loop limit exceeded)$/);
```

Comment:
It appears that the error text is a bit different in Safari and Firefox: 'ResizeObserver loop completed with undelivered notifications.'

We should probably tweak the regex here so that it recognizes that version as well, since we'll likely run these tests in multiple browsers.

Reply:
Added the other text form.  Thanks!